msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -135,7 +140,7 @@ Content-Location: /teams/{teamId}/channels/{channelId}
 
 ## Step Three: Import messages
 
-After the team and channel have been created, you can begin sending back-in-time messages using the `createdDateTime`  and `from`  keys in the request body.
+After the team and channel have been created, you can begin sending back-in-time messages using the `createdDateTime`  and `from`  keys in the request body. **NOTE**: messages imported with `createdDateTime` earlier than the message thread `createdDateTime` is not supported.
 
 #### Request (POST message that is text-only)
```

Comment:
we should add a "Error Response" section for the "Import messages" steps like we do for other steps
@animeshkundu, we can include
- 400 Bad Request - createdDateTime set for future
- 400 Bad Request - impersonated user is in a different tenant from thread's tenant
- 429 Throttled

Reply:
Good call out, I would consolidate the error messages and add them here

Reply:
4xx errors are thrown by TGS in the following cases:
- Teamwork.Migrate.All role is missing in the token
- Creation date time is set in the future in the incoming payload
- Both user identifier or display name is missing in the incoming request.
- The specified user Id in the request payload is not present in the tenant.
- ChatMessage Id is set in the request.
- Body content is missing in the request.
- Reactions are set in the request.
- Weburl is set in the request.